onnxruntime/core/framework/session_state.h

```diff
@@ -143,13 +143,17 @@ class SessionState {
    */
   const std::unordered_map<int, OrtValue>& GetInitializedTensors() const;
 
+  std::unordered_map<int, OrtValue>& GetMutableInitializedTensors();
+
```

Comment:
Doesn't look like there are any external users of this. Is it required? If so, can it be private instead of public? 

In general you want to avoid providing direct non-const access to internals of a class as you lose control of the invariants (i.e. code outside of the class implementation can change the internal state of the object).

Reply:
I don't know how to do these. Can you suggest quick fix for all repos / team members and Immediate team support for the Federal Register Additional Materials for the eve